### PR TITLE
Re-parse / re-render svg tree if default color is changed

### PR DESCRIPTION
**Pull request by brentpicasso**
* allow setting of current_color via a new color property
* this will re-trigger a re-parse and re-render of the XML DOM representing the SVG tree
* some clean up of logging

If an svg file specifies 'currentColor' as a color for stroke or fill, the Svg renderer will attempt to use current_color as a default, if set.  This patch extends this concept by allowing the color property to be updated during runtime, allowing the default color to be changed on the fly.

### CODE REVIEW

**Comment by brentpicasso** [comment]
Example of effect: https://youtu.be/eaYbPqEIpVg

File: kivy/graphics/svg.pyx [hunk 1]
```diff
@@ -568,14 +571,24 @@ cdef class Svg(RenderContext):
             return self._anchor_y
 
 
+    '''Set the default color.
+
+    Used for SvgElements that specify "currentColor"
+    '''
```
Comment (tito) [inline kivy/graphics/svg.pyx]: Can you add the tag:

```
.. versionadded:: 1.9.1
```

?

Thanks you :)

Comment (brentpicasso) [inline kivy/graphics/svg.pyx]: done!

**Comment by tito** [comment]
Thanks!

**Comment by brentpicasso** [comment]
\o/  My pleasure. Thank you for making an amazing framework.

**Pull request merged**